setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(
+    name="imsms_analysis",
+    entry_points={
+        'q2_mlab.models': [
+            'RandomForestPCA=imsms_analysis.custom_models:RandomForestPCA'
+        ]
+    },
+    install_requires=["umap-learn"]
+)
```

Comment:
Does this file indicate that it is required to conda/pip install this package in order to use the RandomForestPCA model?  Why is this a requirement?  Could I not pass you a reference to the imsms_analysis.custom_models:RandomForestSVD class rather than the string "RandomForestSVD"?  This seems to follow a plugin architecture - but the imsms-ml project is less trying to write a plugin than to define and specify an object that meets your required interface.

Reply:
`install_requires=["umap-learn"]` means the imsms-ml module requires it to run. This is unrelated to the entry points but seems to be the required when I tried running analysis.py.